### DISCUSSION TITLE
Only 1 cam out of 5 is connected

### DISCUSSION THREAD

**Discussion by ashtonwalk** [mrlt8/docker-wyze-bridge#1283] [Q&A]
Hello,

I just installed the wyze docker bridge in home assistant for the first time and I am having some issues getting the connection to majority of my cameras. The one cam that is working is my Pan V3 but my 2 cam v4 and 2 cam v3 are not loading. One v3 is missing entirely. I checked the logs but did not see anything that jumped out at me. If someone could point me in the right direction that would be appreciated. All cams and bridge are on the same network. The drop down in the ui shows that the non working cams are not connected?

**Comment by mrlt8** [comment]
hmm, does the missing v3 have the same name as another camera in the wyze app? 

Can you try "reload camera data" from the restart dropdown in the bridge's webUI?

**Reply by ashtonwalk** [reply]
Hello,

Yes the V3 had the same name as another cam. I changed that, thank you for pointing that out. I created a new api key and id today for wyze and updated the bridge but the same issues persist. The 2 cam v4s are intermittently getting snapshot updates but no livestream. The snapshot updates are taking at least an hour or more every time. 

I attempted the reload camera data but nothing changed.

**Reply by mrlt8** [reply]
Are they battery-powered cams or non-supported cams?

**Reply by ashtonwalk** [reply]
they are two cam v4s and two cam v3s. No battery powered cams.
